downloader/src/main/kotlin/VersionControlSystem.kt

```diff
@@ -280,7 +280,12 @@ abstract class VersionControlSystem {
      * Check whether the given [revision] is likely to name a fixed revision that does not move.
      */
     fun isFixedRevision(workingTree: WorkingTree, revision: String) =
-            revision.isNotBlank() && revision !in latestRevisionNames && revision !in workingTree.listRemoteBranches()
+            try {
+                revision.isNotBlank() && revision !in latestRevisionNames &&
+                        revision !in workingTree.listRemoteBranches()
+            } catch (e: IOException) {
+                false
```

Comment:
consider adding a log message in case of an exception with e.g. low severity

Reply:
To be honest, I don't think this is the type of catched exception you'd like to know about...